src/main/kotlin/com/target/liteforjdbc/PreparedStatementSetParameters.kt

```diff
@@ -32,6 +29,64 @@ fun PreparedStatement.setInstant(index: Int, instant: Instant) {
     setObject(index, LocalDateTime.ofInstant(instant, ZoneOffset.UTC))
 }
 
+/**
+ * Sets LocalDateTime parameter
+ */
+fun PreparedStatement.setLocalDateTime(index:Int, value: LocalDateTime) {
+    setObject(index, value)
+}
+
+/**
+ * Sets LocalDate parameter
+ */
+fun PreparedStatement.setLocalDate(index:Int, value: LocalDate) {
+    setObject(index, value)
+}
+
+/**
+ * Sets LocalTime parameter
+ */
+fun PreparedStatement.setLocalTime(index:Int, value: LocalTime) {
+    setObject(index, value)
+}
+
+/**
+ * Sets OffsetDateTime parameter
+ */
+fun PreparedStatement.setOffsetDateTime(index:Int, value: OffsetDateTime) {
+    setObject(index, value)
+}
+
+/**
+ * Sets OffsetTime parameter
+ */
+fun PreparedStatement.setOffsetTime(index:Int, value: OffsetTime) {
+    setObject(index, value)
+}
+
+/**
+ * Sets instant by converting it to LocalDateTime, assuming the timezone is UTC.
```

Comment:
wrong comment?

Reply:
@dtanner yep, and resolved. The change invalidated your approval